https://issues.redhat.com/browse/ACM-20886 (#7870)
open-cluster-management/rhacm-docs: global_hub/global_hub_install_acm.adoc

[#global-hub-install-acm]
= Installing {global-hub} on an existing {acm-short} hub cluster

You can install {global-hub} in an existing {acm-short} hub cluster and enable the {global-hub} agent in {acm-short}. Learn about the following capabilities: 

> * When you install {global-hub} in an {acm-short} hub cluster, Kafka receives this information and stores it in the {global-hub} database, giving you access to the clusters, policies, and events. 

Reviewer: What is "this information"? That you installed or enabled the global hub?

> * Kafka stores this information, so as a Kafka consumer, you can access other components. For example: you can integrate with Red Hat Advanced Cluster Security and Red Hat Event-Driven Ansible. 

Reviewer: What is "this information"?
```suggestion
* Kafka stores this information, so as a Kafka consumer, you can access other components. For example, you can integrate with Red Hat Advanced Cluster Security and Red Hat Event-Driven Ansible. 
```

[#global-hub-install-acm]
= Installing {global-hub} on an existing {acm-short} hub cluster

> You can install {global-hub} in an existing {acm-short} hub cluster and enable the {global-hub} agent in the {acm-short}. By doing this, you have the following capabilities: 

Reviewer 2: We don't need "by doing this" -- extra words.

> You can install {global-hub} in an existing {acm-short} hub cluster and enable the {global-hub} agent in the {acm-short}. By doing this, you have the following capabilities: 

Reviewer 2: ```suggestion
You can install {global-hub} in an existing {acm-short} hub cluster and enable the {global-hub} agent in {acm-short}. Learn more about the capabilities: 
```

Reviewer 2: By doing this, In doing this, because of this....all phrases that can go away for conciseness. The checklist and style guide has some examples, I think.

> * You have access to all the clusters, policies, and events in the {global-hub} database, because when you install {global-hub} in an {acm-short} hub cluster, Kafka received this information and stores it in the database. 

Author: What do you think about making this list more user focused? So we state more so what users can do and have access to after installing global hub on an acm hub cluster rather than just stating what happens when they do it. 

Also, I am cautious to linking to other red hat documentation. do we need to? seems fine to me to simply say that they can integrate with RH ACS and EDA, and we don't have to link there.

Reviewer 3: It's OK for me
/cc @clyang82

Reviewer 4: it's fine for me

> * You have access to all the clusters, policies, and events in the {global-hub} database, because when you install {global-hub} in an {acm-short} hub cluster, Kafka received this information and stores it in the database. 

Reviewer 2: ```suggestion
* You have access to all the clusters, policies, and events in the {global-hub} database. When you install {global-hub} in an {acm-short} hub cluster, Kafka receives this information and stores it in the database. 
```

> * Since Kafka has this information, you can easily access other components that Kafka consume, for example: you can integrate with Red Hat Advanced Cluster Security and Red Hat Event-Driven Ansible. 

Reviewer 2: Instead of **has** and **have**, try to reword these instances.

Reviewer 4: > you can easily access other components that Kafka consume,

should be `it can be easily accessed as a Kafka consumer,

> * Since Kafka has this information, you can easily access other components that Kafka consume, for example: you can integrate with Red Hat Advanced Cluster Security and Red Hat Event-Driven Ansible. 

Reviewer 2: ```suggestion
* Kafka stores this information so that you can easily access other components that Kafka consume, for example: you can integrate with Red Hat Advanced Cluster Security and Red Hat Event-Driven Ansible. 
```

> * Since your data gets stored in the {global-hub} database, you can view the cluster and policy information in the Grafana dashboards. 

Reviewer 2: I have seen a few "gets" in PRs that should be active/present "is"

> * Since your data gets stored in the {global-hub} database, you can view the cluster and policy information in the Grafana dashboards. 

Reviewer 2: ```suggestion
* Your data is stored in the {global-hub} database so that you can view the cluster and policy information in the Grafana dashboards. 
```

> Before you can install {global-hub} in an existing {acm-short} hub cluster, you must create a `MultiClusterGlobalHub` custom resource (CR), by completing the following procedures: 

Author: Am I understanding correctly that users must create a multi cluster global hub CR before installing global hub in an acm hub cluster? and therefore, we're linking to procedures for them to complete? 

If I am correct, I think our "note" is more helpful to users to be put before they start installing the global hub.

Reviewer 3: Yes, you are right.
I think it does not matter to put the `note`, it's just a notification for user.

Reviewer 5: I think this "Before" is not exact here,  the `MultiClusterGlobalHub` custom resource (CR) must be created after you installed global-hub operator.  Actually, It's normal installation steps, we may link the installation procedures.

* xref:../global_hub/global_hub_install_connected.adoc#global-hub-install-connected[Installing Multicluster Global Hub in a connected environment]
* xref:../global_hub/global_hub_install_disconnected.adoc#global-hub-install-disconnected[Installing Multicluster Global Hub in a disconnected environment]

> *Note:* If you enabled the `MultiClusterObservability` CR in the {acm-short}, after you installed {globalhub}, then any new update for the `MultiClusterObservability` CR does not take effect.

Reviewer 2: We don't use CR, I don't think. Please scan the doc, spell out this term.

  availabilityConfig: High
----

The {global-hub} agent is now automatically installed on an existing {acm-short} hub cluster, and you can create a `MultiClusterGlobalHub` custom resource by completing the following procedures: 

* xref:../global_hub/global_hub_install_connected.adoc#global-hub-install-connected[Installing Multicluster Global Hub in a connected environment]
* xref:../global_hub/global_hub_install_disconnected.adoc#global-hub-install-disconnected[Installing Multicluster Global Hub in a disconnected environment]

> *Note:* If you enabled the `MultiClusterObservability` custom resource in the {acm-short}, after you installed {globalhub}, then any new update for the `MultiClusterObservability` custom resource does not take effect.

Reviewer: It's not clear to me what you want to say here. Do you mean that "If you enable the CR after installing global hub, no new updates for the CR will take effect"?
```suggestion
*Note:* If you enabled the `MultiClusterObservability` custom resource in the {acm-short} after you installed {global-hub}, then any new update for the `MultiClusterObservability` custom resource does not take effect.
```
